Fix typo related to Manage APIs Configuration perm. (#200)
mulesoft/docs-api-manager: modules/ROOT/pages/building-https-proxy.adoc

* Secrets Manager Permissions enabled for your user. +
To configure an existing TLS context for the API Proxy, you need the *Read secrets metadata* permission.
Review the xref:anypoint-security::asm-permission-concept.adoc[Secrets Manager Permissions] reference for more information.

> * Make sure your user has the *Manage APIs Configuration* permission for the environment the proxy will be created in.

Reviewer: ```suggestion
* Make sure your user has the *Manage APIs Configuration* permission for the environment in which to create the proxy. 
```

> * A secret group along with a TLS context in Secrets Manager. +

Reviewer: Secrets manager is lc

> Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions along with the  xref:anypoint-security::asm-secret-group-creation-task.adoc#add-a-tls-context[Add a TLS Context] task contained in that article if you need help creating a TLS context. +

Reviewer: For adding a TLS context, the correct xref is:
xref:anypoint-security::secret-group-add-tls-context.adoc#add-tls-context[Add a TLS Context]

> Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions along with the  xref:anypoint-security::asm-secret-group-creation-task.adoc#add-a-tls-context[Add a TLS Context] task contained in that article if you need help creating a TLS context. +

Reviewer: ```suggestion
Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions and xref:anypoint-security::secret-group-add-tls-context.adoc#add-tls-context[Add a TLS Context] instructions if you need help creating a TLS context. +
```
